src/package-tree.ts

```diff
@@ -0,0 +1,65 @@
+interface PointOfInterest {
+  type: string;
+  filename: string;
+  lineNum: number;
+}
+
+export interface PackageTree {
+  rootPackageName: string;
+  version: string;
+  data: PointOfInterest[];
+  dependencies: PackageTree[];
+}
+
+function getDependencies(pjson: string): PackageTree {
+  // todo -- this function will probably be recursive
+  throw new Error('not implemented');
+  // compute result
+  //   let result: PackageTree;
+  //   return result;
+}
+
+function getPOIforPackageTree(packageTree: PackageTree): PackageTree {
+  // TODO:
+  throw new Error('not implemented');
+  // step 1: get POI modules for current package
+  // step 2: add POI to PackageTree Object
+}
+
+
+function getPackagePOIList(package: PackageTree): PointOfInterest[] {
+  // TODO:
+  // calls getPointsOfInterest for each file in package
+  throw new Error('not implemented');
+}
+
+// Small example regarding IO modules instead of POI
+
+// function getIOModules(packageTree: PackageTree<null>):
```

Comment:
Can you remove commented out code? (You can leave it if they are notes for implementation)